src/Contracts/Pools/OpdexLiquidityPool.cs

```diff
@@ -144,6 +145,7 @@ protected UInt256 MintExecute(Address to)
             AmountCrs = amountCrs,
             AmountSrc = amountSrc,
             AmountLpt = liquidity,
+            TotalSupply = TotalSupply,
```

Comment:
Possibly a minor gas saving if `MintTokensExecute` were to return the updated total supply, then believe there will be no need to read from `State` here.

Reply:
While these changes would prevent the extra read, I'm going to stick with the current implementation simply due to the LOE around updating tests and how these low level methods return values are currently being used.